assembly/std/arguments.ts

```diff
@@ -45,8 +45,10 @@ export class Args {
    */
   nextAddress(): Address {
     const length = this.nextU32();
+    const isValid = this.nextBool();
     let address = Address.fromByteArray(
       this.serialized.slice(this.offset as i32, (this.offset as i32) + length),
```

Comment:
Since you changed offset to an `i32`, I think you can remove the `as i32`s from this line. And if there are other instances of `this.offset as i32`, I think the `as i32` can also be removed.

Reply:
good catch, thanks!